README.md

```diff
@@ -27,3 +27,12 @@ Feature with geometry type Polygon and properties Symbol[:geometry, :timestamp,
 # use the Tables interface to convert the format, extract data, or iterate over the rows
 julia> df = DataFrame(fc)
 ```
+
+## HTTP access 
+```julia
```

Comment:
Can you just add a one line description below the heading? Something like "To read JSON from a URL, use HTTP.jl:

Reply:
done